src/lib/components/workflow-actions.svelte

```diff
@@ -145,18 +145,20 @@
     menuLabel={translate('workflows.workflow-actions')}
   >
     {#each workflowActions as { onClick, destructive, label, enabled, testId, description }}
-      {#if destructive}
-        <MenuDivider />
+      {#if !$page.data.settings.runtimeEnvironment.isCloud || ($page.data.settings.runtimeEnvironment.isCloud && testId !== 'update-button')}
```

Comment:
NIT: Could use `isCloud` from `'$lib/stores/advanced-visibility` here instead if you wanted.

Reply:
👍